ibis/backends/base/sql/compiler/query_builder.py

```diff
@@ -41,12 +41,19 @@ def __init__(self, parent, node, indent=2):
         self.indent = indent
 
```

Comment:
do we want to have these changes in the base SQL compiler implementation, or just override it in FlinkSQL backend's compiler implementation?